packages/expo-linking/ios/ExpoLinkingAppLifecycleDelegate.swift

```diff
@@ -0,0 +1,11 @@
+import ExpoModulesCore
+
+public class ExpoLinkingAppLifecycleDelegate: ExpoAppDelegateSubscriber {
```

Comment:
```suggestion
public class LinkingAppDelegateSubscriber: ExpoAppDelegateSubscriber {
```

Reply:
![image](https://github.com/expo/expo/assets/5597580/d9bb51d9-8fce-4c31-b79e-c1b26a482ced)

I think all the others are named like that

Reply:
There is also `ScreenOrientationAppDelegate` so we're not very consistent 😅 Anyway, let's not use `AppLifecycle`.

Reply:
oh, sorry, thought you meant the expo prefix, fixing